ingest/src/main/java/com/microsoft/azure/kusto/ingest/result/IngestionStatusInTableDescription.java

```diff
@@ -46,6 +49,14 @@ public TableClient getTableClient() {
         return tableClient;
     }
 
+    public void setTableUri(String tableUri) {
```

Comment:
you dont need all these 
use the connection string (add a getTableConnectionString)